### PR TITLE
Update dlp-chrome-get-started.md

### PR DESCRIPTION
**Pull request by jperhamcatchteam**
Updated configuration profile steps for "Deploy using Microsoft Endpoint Manager" to match the current Microsoft Endpoint Manager admin center device configuration policy creation flow.

### CODE REVIEW

**Comment by jperhamcatchteam** [comment]
@microsoft-github-policy-service agree

**Comment by jperhamcatchteam** [comment]
@microsoft-github-policy-service agree

File: microsoft-365/compliance/dlp-chrome-get-started.md [hunk 1]
```diff
@@ -140,21 +140,33 @@ Before adding the extension to the list of force-installed extensions, it is imp
...
+4. Select **Windows 10 and later** as the platform.
 
-5. Select **Custom** as profile type.
+5. Select **Templates** as the profile type.
 
-6. Select the **Settings** tab.
+6. Select **Custom** as the template name.
 
-7. Select **Add**.
+7. Click create.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-chrome-get-started.md]:
```suggestion
7. Select **Create**.
```

File: microsoft-365/compliance/dlp-chrome-get-started.md [hunk 2]
```diff
@@ -140,21 +140,33 @@ Before adding the extension to the list of force-installed extensions, it is imp
...
 
-8. Enter the following policy information.
+8. Enter a name and optional description on the **Basics** tab and click next.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-chrome-get-started.md]:
```suggestion
8. Enter a name and optional description on the **Basics** tab and select **Next**.
```

File: microsoft-365/compliance/dlp-chrome-get-started.md [hunk 3]
```diff
@@ -140,21 +140,33 @@ Before adding the extension to the list of force-installed extensions, it is imp
...
+
+9. Select **Add** on the **Configuration settings** tab.
+
+10. Enter the following policy information.
 
     OMA-URI: `./Device/Vendor/MSFT/Policy/Config/Chrome~Policy~googlechrome~Extensions/ExtensionInstallForcelist`<br/>
     Data type: `String`<br/>
     Value: `<enabled/><data id="ExtensionInstallForcelistDesc" value="1&#xF000; echcggldkblhodogklpincgchnpgcdco;https://clients2.google.com/service/update2/crx"/>`
+    
+11. Click save and click next.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-chrome-get-started.md]:
```suggestion
11. Select **Save** and then select **Next**.
```

File: microsoft-365/compliance/dlp-chrome-get-started.md [hunk 4]
```diff
@@ -140,21 +140,33 @@ Before adding the extension to the list of force-installed extensions, it is imp
...
+
+12. Add or edit scope tags on the **Scope tags** tab if desired and click next.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-chrome-get-started.md]:
```suggestion
12. Add or edit scope tags on the **Scope tags** tab as needed and select **Next**.
```

File: microsoft-365/compliance/dlp-chrome-get-started.md [hunk 5]
```diff
@@ -140,21 +140,33 @@ Before adding the extension to the list of force-installed extensions, it is imp
...
+
+13. Add the desired deployment users/devices/groups on the **Assignments** tab and click next.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-chrome-get-started.md]:
```suggestion
13. Add the required deployment users, devices, and groups on the **Assignments** tab and select **Next**.
```

File: microsoft-365/compliance/dlp-chrome-get-started.md [hunk 6]
```diff
@@ -140,21 +140,33 @@ Before adding the extension to the list of force-installed extensions, it is imp
...
+
+14. Add applicability rules on the **Applicability Rules** tab if desired and click next.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-chrome-get-started.md]:
```suggestion
14. Add applicability rules on the **Applicability Rules** tab as required and select **Next**.
```

File: microsoft-365/compliance/dlp-chrome-get-started.md [hunk 7]
```diff
@@ -140,21 +140,33 @@ Before adding the extension to the list of force-installed extensions, it is imp
...
 
-9. Click create.
+15. Click create.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-chrome-get-started.md]:
```suggestion
15. Select **Create**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @jperhamcatchteam, kindly commit the suggested changes. Many thanks.

**Comment by jperhamcatchteam** [comment]
> Copy review - @jperhamcatchteam, kindly commit the suggested changes. Many thanks.

Done.  Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Comment by chrfox** [comment]
confirming with PM

**Pull request merged**